addons/website/views/snippets/s_product_catalog.xml

```diff
@@ -56,6 +56,20 @@
 </template>
 
 <template id="s_product_catalog_options" inherit_id="website.snippet_options">
+    <xpath expr="//t[@t-call='web_editor.snippet_options_background_options']" position="before">
+        <div data-js="MultipleItems" data-selector=".s_product_catalog .row > div">
+            <we-row string="Product">
+                <we-button title="Add Product"
+                    data-add-item=""
+                    data-item=".s_product_catalog_dish:last"
+                    data-select-item=""
+                    data-no-preview="true"
+                    data-apply-to=":has(.s_product_catalog_dish)"
```

Comment:
Yes, I used the "apply-to" to prevent the option from appearing when the target column in Pricelist is empty, since the "add item" options are based on cloning existing elements and not creating new ones...

Reply:
My point is that `:has(.hello)`, in that example:

```xml
<div><section><span><i class="hello"/></span></section></div>
```
matches the div, the section and the span and I don't think we ever will want that.